### ISSUE TITLE
Add google analytics

### ISSUE THREAD

**Issue by ameer-clara** [codiak/socialarchive-dapp#36]

**Comment by codiak** [comment]
Is there a privacy focused alternative to GA we could use?

**Comment by ameer-clara** [comment]
Good point, we need the European style GDPR tracking cookie warning modal on first visit, for now UA is the best, we should keep alternatives in mind and revisit this.
